lua/lspconfig/configs/denols.lua

```diff
@@ -96,7 +96,7 @@ return {
   commands = {
     DenolsCache = {
       function()
-        local clients = util.get_lsp_clients { bufnr = 0, name = 'denols' }
+        local clients = util.get_lsp_clients({ bufnr = 0, name = 'denols' })
```

Comment:
is stylua doing this? not required by our style guide.

Reply:
seems like a bug of stylua . just run `stylua ./` in shell